ai_bridge/__manifest__.py

```diff
@@ -0,0 +1,25 @@
+# Copyright 2025 Dixmit
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+{
+    "name": "Ai Bridge",
+    "summary": """Makes a basic configuration to be used as bridge with external AI systems""",
+    "version": "16.0.1.0.0",
+    "license": "AGPL-3",
+    "author": "Dixmit,Odoo Community Association (OCA)",
+    "website": "https://github.com/OCA/ai",
+    "depends": ["mail"],
```

Comment:
please add 
```
development_status": "Beta
```
https://github.com/OCA/odoo-community.org/blob/master/website/Contribution/oca_module_lifecycle_development_status.rst#2alpha-and-beta-modules

Reply:
Alpha or Beta?

Reply:
I think **Beta** -> Pre-production quality but with potential instability .

Documentation:

- The **Alpha** stage is not required, but it may be useful to signal that the module is not expected to work as planned yet (or at all). It is in the early stages of design and construction, and trying it on Runboat is pointless

- Changing the development status to **Beta** is a way to signal reviewers and functional testers that the module should be usable enough to be tested.